modules/ocl-loader/src/main/java/org/eclipse/xpanse/modules/service/CreateRequest.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ *
+ */
+
+package org.eclipse.xpanse.modules.service;
+
+import io.swagger.v3.oas.annotations.Hidden;
+import jakarta.persistence.CollectionTable;
+import jakarta.persistence.Column;
+import jakarta.persistence.ElementCollection;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.MapKeyColumn;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+import lombok.Data;
+import org.eclipse.xpanse.modules.ocl.loader.data.models.enums.Csp;
+
+/**
+ * Request body for service creation.
+ */
+@Data
+public class CreateRequest {
+
+    @Hidden
+    @Id
+    private UUID id;
+
+    /**
+     * The name of the Service.
+     */
+    @NotNull
+    @NotBlank
+    String name;
+
+    /**
+     * The version of the Service.
+     */
+    @NotNull
+    @NotBlank
+    String version;
+
+    /**
+     * The csp of the Service.
+     */
+    @NotNull
+    Csp csp;
+
+    /**
+     * The flavor of the Service.
+     */
+    @NotNull
+    @NotBlank
+    String flavor;
+
+    /**
+     * The property of the Service.
+     */
+    @ElementCollection

```

Comment:
is this bean also a DB entity?

Reply:
no, let me delete these database annotations.